src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -308,21 +308,271 @@ describe('CdkOption', () => {
       expect(listboxInstance._listKeyManager.activeItem).toEqual(optionInstances[2]);
       expect(listboxInstance._listKeyManager.activeItemIndex).toBe(2);
     });
+
+    it('should update selected option on click event', () => {
+      let selectedOptions = optionInstances.filter(option => option.selected);
+
+      expect(selectedOptions.length).toBe(0);
+      expect(optionElements[0].hasAttribute('aria-selected')).toBeFalse();
+      expect(optionInstances[0].selected).toBeFalse();
+      expect(fixture.componentInstance.changedOption).toBeUndefined();
+
+      dispatchMouseEvent(optionElements[0], 'click');
+      fixture.detectChanges();
+
+      selectedOptions = optionInstances.filter(option => option.selected);
+      expect(selectedOptions.length).toBe(1);
+      expect(optionElements[0].getAttribute('aria-selected')).toBe('true');
+      expect(optionInstances[0].selected).toBeTrue();
+      expect(fixture.componentInstance.changedOption).toBeDefined();
+      expect(fixture.componentInstance.changedOption.id).toBe(optionInstances[0].id);
+    });
+  });
+
+  describe('multiselectable tests', () => {
```

Comment:
```suggestion
  describe('with multiple selection', () => {
```
Prefer naming `describe` sub-blocks like this so that the end result reads like a sentence, e.g.
```
listbox with muliple selection should select all options using the select all method
```

Reply:
I see, noted.